Sasquatch/SasquatchObjC/AppCenterDelegateObjC.m

```diff
@@ -70,6 +70,10 @@ - (void)startAnalyticsFromLibrary {
   [MSAppCenter startFromLibraryWithServices:@[ [MSAnalytics class] ]];
 }
 
+- (void)setUserId:(NSString *)userId {
+  [MSAppCenter setUserId:userId];
```

Comment:
Since I couldn't see userId in payload after testing against latest version of base branch (locally merged), I put a breakpoint here when testing via SasquatchPuppet and it's never triggered. We don't seem to call setUserId from UI.

Reply:
Agrh! `valueChanged` vs `editingChanged` 😄